src/command.cpp

```diff
@@ -298,8 +298,8 @@ bool Command::set_attribute(JsonObject output, const char * cmd, const char * at
     if (attribute == nullptr) {
         return true;
     }
-    if (output.containsKey(attribute)) {
-        std::string data = output[attribute].as<std::string>();
+    if (output[attribute].is<std::string>()) {
```

Comment:
Does this work? The attribute "value" can also be float or int, "writable" is bool, etc. We force it to read as string for output to "api_data".

Reply:
I change to JsonVariantConst. Thanks

Reply:
all the unit tests pass so I think its ok